scene/2d/tile_map_layer.h

```diff
@@ -425,6 +425,7 @@ class TileMapLayer : public Node2D {
 	// --- Cells manipulation ---
 	// Generic cells manipulations and data access.
 	void set_cell(const Vector2i &p_coords, int p_source_id = TileSet::INVALID_SOURCE, const Vector2i &p_atlas_coords = TileSetSource::INVALID_ATLAS_COORDS, int p_alternative_tile = 0);
+	void place_random_tiles(Array atlas_coords_array, Rect2i region, float scattering_probability, bool replace_tile);
```

Comment:
```suggestion
	void place_random_tiles(const TypedArray<Vector4i> &p_atlas_coords_array, const Rect2i &p_region, float p_scattering_probability, bool p_replace_tile);
```
Prefix with `p_`, pass by reference, and add static typing to the argument